oauth2_http/java/com/google/auth/http/AuthHttpConstants.java

```diff
@@ -0,0 +1,18 @@
+package com.google.auth.http;
+
+import java.util.regex.Pattern;
+
+/**
+ * Constants used for auth in http
+ */
+public class AuthHttpConstants {
```

Comment:
Plan of action here:
* leave "Authorization" here
* make a constant "Bearer" here
* Make a package-private constant for "Bearer" + " " in each scope it is needed
* Move INVALID_TOKEN_ERROR to where it is used.